torchvision/csrc/models/densenet.h

```diff
@@ -23,10 +23,10 @@ struct VISION_API DenseNetImpl : torch::nn::Module {
   torch::nn::Sequential features{nullptr};
   torch::nn::Linear classifier{nullptr};
 
-  DenseNetImpl(
+  explicit DenseNetImpl(
```

Comment:
for my understanding: is this the default recommended way of constructing models in C++?

Note that we do not use `explicit` in our other model implementations.

cc @anjali411 do you have any recommendation here?

Reply:
@fmassa Thanks for flagging it. As discussed on call, it's probably better to drop this and revisit the static analysis recommendation when I'm more certain about the implications of this. :)